app/pages/home/homePageController.ts

```diff
@@ -25,22 +28,30 @@ export class HomePageController {
   };
 
   public update() {
+
+    // TODO: Remove these try/catches once we are not relying on 'additionalServers' in config.js
```

Comment:
@jwforres I think unrecognized resource should be a promise failure instead of runtime error. Then if the service catalog is down or not installed, it can use our existing error handling.